README.md

```diff
@@ -1,6 +1,18 @@
 # terraform-ap-modules
 DSP Analysis Platform Terraform Modules. This public repository contains Terraform modules that are meant to be Broad-agnostic and can potentially be deployed by outside parties. The general pattern for their deployment inside the Broad is for them to be referenced from a deployment in the private [terraform-ap-deployments repository](https://github.com/broadinstitute/terraform-ap-deployments), along with any Broad-specific infrastructure.
 
+## Workflow for Atlantis-managed deployments
+When making changes to modules deployed with Atlantis in the [terraform-ap-deployments repository](https://github.com/broadinstitute/terraform-ap-deployments), the following workflow must be followed:
+1. Create a PR in this repo with changes to the module(s). If changing a module that is a sub-module of another module (such as `terra-env`), remember to point the module reference to the PR branch.
+2. Create a PR in terraform-ap-deployments with the module reference of the changing module pointing to the modules repo PR branch.
+3. [Run a plan](https://github.com/broadinstitute/terraform-ap-deployments#3-iterate-using-atlantis) in the ap-deployments PR. If changing multiple environments/projects it is recommended to initially reach a green plan/apply for one of them before changing the others.
```

Comment:
Maybe include an example of how to run a plan for a specific environment here for quick reference?

Reply:
makes sense

Reply:
Also - worth noting you cannot run an apply without review - I might move the "one env" recommendation to step #5.